realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -353,6 +380,27 @@ public Builder(Context context) {
             initializeBuilder(context.getFilesDir());
         }
 
+        /**
+         * Creates an instance of the Builder for the RealmConfiguration.
+         *
+         * This will use the app's own internal directory for storing the Realm file. This does not require any
+         * additional permissions. The default location is {@code /data/data/<packagename>/files}, but can
+         * change depending on vendor implementations of Android.
+         *
+         * Initial dataset will be imported from the provided Realm asset file.
+         *
+         * @param context Android context.
```

Comment:
I would call this "Android application context", just so it is more clear that you shouldn't use the Activity context.